l10n_br_sale_product/sale.py

```diff
@@ -251,6 +261,12 @@ def onchange_amount_costs(self, cr, uid, ids, amount_costs=False):
 class SaleOrderLine(orm.Model):
     _inherit = 'sale.order.line'
 
+    def _calc_line_base_price(self, cr, uid, line, context=None):
+        return line.price_unit * (1 - (line.discount or 0.0) / 100.0)
```

Comment:
Yes, I agree with you. However we will have to wait a few years for the Baeza's pr be approved ....